native/src/routes/events/components/Events.js

```diff
@@ -63,39 +57,31 @@ class Events extends React.Component<PropsType> {
   }
 
   createNavigateToFeedbackForEvent = (event: EventModel) => (isPositiveFeedback: boolean) => {
-    const { t, navigation, cities, cityCode, language } = this.props
-
-    const createFeedbackVariant = (
-      label: string, feedbackType: FeedbackType, feedbackCategory: FeedbackCategoryType, pagePath?: string
-    ): FeedbackVariant => new FeedbackVariant(label, language, cityCode, feedbackType, feedbackCategory, pagePath)
-
-    const cityTitle = CityModel.findCityName(cities, cityCode)
-    navigation.navigate('FeedbackModal', {
-      isPositiveFeedback,
-      feedbackItems: [
-        createFeedbackVariant(t('feedback:contentOfEvent', { event: event.title }), PAGE_FEEDBACK_TYPE,
-          CONTENT_FEEDBACK_CATEGORY, event.path),
-        createFeedbackVariant(t('feedback:contentOfCity', { city: cityTitle }), EVENTS_FEEDBACK_TYPE,
-          CONTENT_FEEDBACK_CATEGORY),
-        createFeedbackVariant(t('feedback:technicalTopics'), CATEGORIES_FEEDBACK_TYPE, TECHNICAL_FEEDBACK_CATEGORY)
-      ]
-    })
+    const { navigation, cityCode, language } = this.props
+
+    const feedbackInformation: FeedbackInformationType = {
+      type: 'Event',
+      title: event.title,
+      path: event.path,
+      cityCode,
+      language,
+      isPositiveFeedback
+    }
+
+    navigation.navigate('FeedbackModal', { ...feedbackInformation })
   }
 
   navigateToFeedbackForEvents = (isPositiveFeedback: boolean) => {
-    const { t, navigation, cities, cityCode, language } = this.props
-    const createFeedbackVariant = (label: string, feedbackType: FeedbackType, feedbackCategory: FeedbackCategoryType,
-      pagePath?: string) =>
-      new FeedbackVariant(label, language, cityCode, feedbackType, feedbackCategory, pagePath)
-    const cityTitle = CityModel.findCityName(cities, cityCode)
-    navigation.navigate('FeedbackModal', {
-      isPositiveFeedback,
-      feedbackItems: [
-        createFeedbackVariant(t('feedback:contentOfCity', { city: cityTitle }), EVENTS_FEEDBACK_TYPE,
-          CONTENT_FEEDBACK_CATEGORY),
-        createFeedbackVariant(t('feedback:technicalTopics'), CATEGORIES_FEEDBACK_TYPE, TECHNICAL_FEEDBACK_CATEGORY)
-      ]
-    })
+    const { navigation, cityCode, language } = this.props
+
+    const feedbackInformation: FeedbackInformationType = {
+      type: 'Event',
```

Comment:
Is the type `Event` correct here for multiple events?

Reply:
I think it should be `Events`, right? https://github.com/Integreat/cms/wiki/REST-APIv3-Documentation#events

Reply:
the type here is not the type that is send to the feedback endpoint. It is just loosely used instead of CATEGORIES_ROUTE, EVENT_ROUTE, ... in web